plugin/sg.lua

```diff
@@ -108,12 +108,11 @@ end, {})
 --- Get a sourcegraph link to the current repo + file + line.
 --- Automatically adds it to your '+' register
 ---@command ]]
-vim.api.nvim_create_user_command("SourcegraphLink", function()
-  local cursor = vim.api.nvim_win_get_cursor(0)
+vim.api.nvim_create_user_command("SourcegraphLink", function(command)
   void(function()
     print "requesting link..."
 
-    local err, link = require("sg.rpc").get_link(vim.api.nvim_buf_get_name(0), cursor[1], cursor[2] + 1)
+    local err, link = require("sg.rpc").get_link(vim.api.nvim_buf_get_name(0), command.line1, command.line2)
```

Comment:
does this still work without doing it in visual mode?  I'd also still like to do the cursor position (so we should just maybe send a whole Range over instead of just a position).

We'll also have to bump the version if we do this to regenerate the releases

Reply:
like we could do `(bufname, strt_line, start_col, end_line?, end_col?)` and it does a range the last two are passed, otherwise it just does the line

Reply:
We lose the column information with this, right? So a visual selection of just a function argument will generate a link to the whole line in sourcegraph